deploy/helm/sumologic/templates/logs/collector/otelcol-cloudwatch/statefulset.yaml

```diff
@@ -0,0 +1,115 @@
+{{- if .Values.sumologic.logs.collector.otelcloudwatch.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "sumologic.metadata.name.logs.collector.statefulset" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.logs.collector.statefulset" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sumologic.labels.app.logs.collector.statefulset" . }}
+  serviceName: {{ template "sumologic.labels.app.logs.cloudwatch.service-headless" . }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/logs/collector/otelcol/configmap.yaml") . | sha256sum }}
+{{- if .Values.sumologic.podAnnotations }}
+{{ toYaml .Values.sumologic.podAnnotations | indent 8 }}
+{{- end }}
+{{- if .Values.otellogs.statefulset.podAnnotations }}
+{{ toYaml .Values.otellogs.statefulset.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        app: {{ template "sumologic.labels.app.logs.collector.statefulset" . }}
+        {{- include "sumologic.labels.common" . | nindent 8 }}
+{{- if .Values.sumologic.podLabels }}
+{{ toYaml .Values.sumologic.podLabels | indent 8 }}
+{{- end }}
+{{- if .Values.otellogs.statefulset.podLabels }}
+{{ toYaml .Values.otellogs.statefulset.podLabels | indent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ template "sumologic.metadata.name.logs.collector.serviceaccount" . }}
+{{- if .Values.otellogs.statefulset.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.otellogs.statefulset.nodeSelector | indent 8 }}
+{{- end }}
+{{- if .Values.otellogs.statefulset.tolerations }}
+      tolerations:
+{{ toYaml .Values.otellogs.statefulset.tolerations | indent 8 }}
+{{- end }}
+{{- if .Values.otellogs.statefulset.affinity }}
+      affinity:
+{{- if .Values.otellogs.statefulset.affinity }}
+{{ toYaml .Values.otellogs.statefulset.affinity | indent 8 }}
+{{- end }}
+{{- end }}
+{{- if .Values.otellogs.statefulset.topologySpreadConstraints }}
+      topologySpreadConstraints:
+{{ toYaml .Values.otellogs.statefulset.topologySpreadConstraints | indent 6 }}
+{{- end }}
+      volumes:
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: config.yaml
+            path: config.yaml
+          name: {{ template "sumologic.metadata.name.logs.collector.configmap" . }}
+        name: otelcol-config
+      - name: file-storage
+        persistentVolumeClaim:
+          claimName: file-storage-sumo-sumologic-otelcol-logs-0
```

Comment:
It should be optional

Reply:
I agree.
Additional comment: currently you don't mention in the docs that these volumes have to be created in the same way as for metrics.